lib/extensions.js

```diff
@@ -260,6 +261,80 @@ function getBinaryPath() {
 }
 
 /**
+ * An array of paths suitable for use as a local disk cache of the binding.
+ *
+ * @return {[]String} an array of paths
+ * @api public
+ */
+function getCachePathCandidates() {
+  return [
+    process.env.npm_config_sass_binary_cache,
```

Comment:
Nevermind, saw that you're looping through later, so you want all truethy values